README.md

```diff
@@ -12,7 +12,7 @@ Wechaty is a Bot Framework for Wechat **Personal** Account which can help you cr
 
 :octocat: <https://github.com/chatie/wechaty>  
 :beetle: <https://github.com/chatie/wechaty/issues>  
-:book: <https://github.com/chatie/wechaty/wiki>  
+:book: <https://github.com/Chatie/wechaty/blob/master/docs/index.md>  
```

Comment:
Please use the url <http://chatie.io/wechaty/> instead of https://github.com/Chatie/wechaty/blob/master/docs/index.md> at here and all of the following links in the document.

Because <http://chatie.io/wechaty/> is the custom url of <https://github.com/Chatie/wechaty/blob/master/docs/>